samples/CSharp/AzureFunctionsRabbitMQSamples/example.settings.json

```diff
@@ -0,0 +1,8 @@
+{
+  "IsEncrypted": false,
+  "Values": {
+    "AzureWebJobsStorage": "UseDevelopmentStorage=true",
+    "FUNCTIONS_WORKER_RUNTIME": "dotnet",
```

Comment:
Please create a folder 'Azure.Functions.Samples' under Samples folder and then sub directories per language:  `CSharp` , `Python` etc